spec/memfs/file_spec.rb

```diff
@@ -950,6 +950,30 @@ module MemFs
       end
     end
 
+    describe '.write' do
+      it 'writes the string to the given file' do
+        described_class.write('/test-file', "test")
+        read_content = described_class.read('/test-file')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.